src/adder.rs

```diff
@@ -108,6 +124,25 @@ fn add_in_generic_value(generic_value: &mut Value, last_field: &str, toml: Item)
     }
 }
 
+fn handle_add_with_table_header_path(
+    table_header_path: &str,
+    dotted_path: &str,
+    value: &str,
+    doc: &mut DocumentMut) -> Result<()> {
```

Comment:
same as the prior nit
```suggestion
    doc: &mut DocumentMut,
) -> Result<()> {
```